ports/llvm-13/0024-remove-elf_relocation-checks.patch

```diff
@@ -0,0 +1,114 @@
+diff --git a/llvm/lib/ExecutionEngine/RuntimeDyld/RuntimeDyldELF.cpp b/llvm/lib/ExecutionEngine/RuntimeDyld/RuntimeDyldELF.cpp
+index efe0b9cd61cd..8d1f2fa1ccc1 100644
+--- a/llvm/lib/ExecutionEngine/RuntimeDyld/RuntimeDyldELF.cpp
++++ b/llvm/lib/ExecutionEngine/RuntimeDyld/RuntimeDyldELF.cpp
+@@ -21,6 +21,7 @@
+ #include "llvm/Object/ObjectFile.h"
+ #include "llvm/Support/Endian.h"
+ #include "llvm/Support/MemoryBuffer.h"
++#include "llvm/Support/raw_ostream.h"
+ 
+ using namespace llvm;
+ using namespace llvm::object;
+@@ -28,6 +29,17 @@ using namespace llvm::support::endian;
+ 
+ #define DEBUG_TYPE "dyld"
+ 
++static void report_reallocation_error(uint32_t ty) {
++  std::string message;
++  llvm::raw_string_ostream ss(message);
++  ss << "Relocation type not implemented yet: " << ty << "!";
++  if (RuntimeDyldELF::ShouldFailOnRelocationErrors) {
++    report_fatal_error(message);
++  } else {
++    dbgs() << message;
```

Comment:
Lets wrap this with `LLVM_DEBUG()` so it would be like other debug statements, which means we could control it via `-debug` on the command line (or manually setting llvm::DebugFlag via gdb/llb/our own commandline/etc)

Reply:
herm in this case we will only be able to see relocation errors in debug builds of llvm which is why I didnt do this, is this desirable?